release-scripts/generate-release-notes.ipynb

```diff
@@ -0,0 +1,714 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "def update_quarto_yaml(release_date):\n",
+    "    \"\"\"Updates the _quarto.yml file to include the release notes file so it can be accessed on the website.\n",
+    "\n",
+    "    Params:\n",
+    "        release_date - release notes use the release date as the file name.\n",
+    "    \n",
+    "    Modifies:\n",
+    "        _quarto.yml file\n",
+    "    \"\"\"\n",
+    "    yaml_filename = \"../site/_quarto.yml\"\n",
+    "    temp_yaml_filename = \"../site/_quarto_temp.yml\"\n",
+    "\n",
+    "    # Copy the original YAML file to a temporary file\n",
+    "    shutil.copyfile(yaml_filename, temp_yaml_filename)\n",
+    "\n",
+    "    with open(temp_yaml_filename, 'r') as file:\n",
+    "        lines = file.readlines()\n",
+    "\n",
+    "    # Format the release date for insertion into the YAML file\n",
+    "    formatted_release_date = release_date.strftime(\"%Y-%b-%d\").lower()\n",
+    "\n",
+    "    with open(yaml_filename, 'w') as file:\n",
+    "        add_release_content = False\n",
+    "        insert_index = -1\n",
+    "\n",
+    "        for i, line in enumerate(lines):\n",
+    "            file.write(line)\n",
+    "            if line.strip() == \"# MAKE-RELEASE-NOTES-EMBED-MARKER\":\n",
+    "                add_release_content = True\n",
+    "                insert_index = i\n",
+    "\n",
+    "            if add_release_content and i == insert_index:\n",
+    "                file.write(f'        - releases/{formatted_release_date}/release-notes.qmd\\n')\n",
+    "                add_release_content = False\n",
+    "\n",
+    "    # Remove the temporary file\n",
+    "    os.remove(temp_yaml_filename)\n",
+    "    \n",
+    "    print(f\"Added release notes to _quarto.yml, line {insert_index + 2}\")\n",
+    "\n",
+    "update_quarto_yaml(release_datetime)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## <a id='toc19_'></a>Show files to commit [](#toc0_)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# After completing all tasks, print git status to show output files\n",
+    "try:\n",
+    "    result = subprocess.run([\"git\", \"status\", \"--short\"], check=True, text=True, capture_output=True)\n",
+    "    lines = result.stdout.split('\\n')\n",
+    "    print(\"Files to commit:\")\n",
+    "    for line in lines:\n",
+    "        if line.startswith((' M', '??', 'A ')):\n",
+    "            print(line)\n",
+    "except subprocess.CalledProcessError as e:\n",
+    "    print(\"Failed to run git status:\", e)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## <a id='toc20_'></a>Preview and edit changes [](#toc0_)\n",
```

Comment:
Can you add a short "Next steps" section that tells you what to do with the release notes, i.e. commit the release notes and start a PR?